.idea/compiler.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do i need to mark ".idea" folder as "gitignor" on my computer and delete this folder on git?